.github/workflows/validation.yml

```diff
@@ -0,0 +1,50 @@
+name: Validation
```

Comment:
we might wanna use the validation we talked about earlier using the action itself. ([link here](https://github.com/gagoar/use-herald-action/blob/master/.github/workflows/use_action_for_build_test.yml))